GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -47,9 +61,36 @@
 // Expected path in the URL scheme to be handled.
 static NSString *const kBrowserCallbackPath = @"/oauth2callback";
 
+// The error code for Google Identity.
+NSErrorDomain const kGIDVerifyErrorDomain = @"com.google.GIDVerify";
```

Comment:
What do you think about keeping this as `com.google.GIDSignIn`? (FWIW You can read more about [error domains here](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ErrorHandlingCocoa/ErrorObjectsDomains/ErrorObjectsDomains.html#//apple_ref/doc/uid/TP40001806-CH202-CJBGAIBJ).)

If we will keep the existing domain (which should get moved to the constants file from #401), then `com.google.GIDSignIn` is the domain for all errors from this library. That means the specific error codes will identify the error. One weakness I see with this approach is that the error codes are just integer backed enum cases, so we could have some collision across errors if two error enum cases are backed by the same integer.

What do you think?

Reply:
I also thought of keeping it as `com.google.GIDSignIn` since I liked the idea of the domain covering the library as a whole but felt being more specific on the error domain wouldn't hurt. Though, I feel that keeping the domain as is would work best if we're able to keep the integer ranges of both enum cases separate. For example, the verify error codes could start at 1000.

Reply:
We're in a kind of tricky spot since `GIDSignInErrorCode`s are negative. :/ 

I think maybe it's better to (for now) to go with `com.google.GIDVerifyAccountDetail` for the domain. It's cleaner that way, IMHO.

If you agree, would you mind creating an issue to revisit and finalize our position on this and link here in the code?